docs/external-runtime-deps.md

```diff
@@ -150,6 +150,15 @@ Optional cgroup controllers:
 [cgroup v1]: https://www.kernel.org/doc/html/v5.16/admin-guide/cgroup-v1/
 [cgroup v2]: https://www.kernel.org/doc/html/v5.16/admin-guide/cgroup-v2.html
 
+### containerd and AppArmor
+
+In order to use containerd in conjunction with [AppArmor], the
```

Comment:
It could be worth mentioning both invariants: the kernel needs AppArmor enabled (compiled in, on-by-default or `apparmor=1`), but then again, for any platform that actually supports AppArmor (Debian-based) they ship AppArmor-enabled kernels out of the box.